src/Classes/DKP_Entry.lua

```diff
@@ -143,50 +143,42 @@ function DKP_Entry:GetFormattedNames()
         end
     end
 
-    local formattedNames = ''
-
-    local function compare(a,b)
-        local a_member = Guild:GetMemberByName(a)
-        local b_member = Guild:GetMemberByName(b)
-
-        if a_member == nil then
-            return true
-        elseif b_member == nil then
-            return false
-        end
-
-        if a_member == nil or b_member == nil then
-            print('ERROR sorting entry names: ', a, b)
-            print(a_member.name, b_member.name)
-            return false
+    local memberTable = {}
+    for key, name in ipairs(self.names) do
+        local member = Guild:GetMemberByName(name)
+        if member ~= nil then
+            table.insert(memberTable, member)
```

Comment:
Creating an array of only valid member objects here. Also has the added benefit of cutting down on the number of times we have to resolve a name to a guild member object.